test-data/unit/check-fastparse.test

```diff
@@ -322,7 +322,7 @@ x @= 1
 from typing import Dict
 x = None # type: Dict[x: y]
 [out]
-main:3: error: syntax error in type comment
+main:3: error: Slice usage in type annotation is invalid, consider using "slice()" or "{}"
```

Comment:
Bikeshedding: What about changing this to "Slice syntax in ...". 

In this context suggesting `slice()` or `{}` is a little questionable, since they aren't supported either. It may be better to drop the "consider using ..." part to avoid confusion unless there's a way to restrict the final part of the message to `Annotated[...]` types somehow.

Reply:
Agreed.